wagtailio/project_styleguide/templates/patterns/components/search.html

```diff
@@ -0,0 +1,47 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags %}
+
+{% block content %}
+    <div class="grid">
+        <div class="hero hero--search">
+            <div class="hero__inner">
+                <h1 class="hero__heading">Search results</h1>
+
+                <div>
```

Comment:
question: i'm wondering if this div can be removed?

Reply:
Yep great catch! I've removed this now.